mtp/mtp_fs.cpp

```diff
@@ -58,10 +64,10 @@ namespace
     const mtp_storage_properties_t *get_disk_properties(void *arg)
     {
         const auto fs = static_cast<struct mtp_fs *>(arg);
-        struct statvfs stvfs {};
-        const auto ret = statvfs(fs->root, &stvfs);
+        struct statvfs stvfs
+        {};
```

Comment:
strange formatting

Reply:
It is automatically formatted this way AFAIK ;)